docker-compose.get-latest.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+function upgrade() {
+  echo "[+] Deleting all containers" && \
+  docker-compose down && \
```

Comment:
this uses the default docker-compose.yaml, but what about the rest of the extra services yaml, e.g., the routinator one? Same for pulling, etc. Maybe the user should specify which yamls he is using somewhere in the script?

Reply:
I think we should make the users to have everything stopped already before upgrading.. so we just start the database->backup->stop->upgrade->restore

Reply:
Is there a command that stops all running instances, whatever the used yaml? Sth like force compose down whatever the "-f ..." used.

Reply:
nope

Reply:
Ok then just add an instruction to the users to shut down their running docker instances before proceeding (and leave also the default `docker-compose down` as you have it right now).